taipy/gui/gui.py

```diff
@@ -25,8 +25,8 @@
 
 import __main__
 import markdown as md_lib
-from flask import Blueprint, Flask, request, send_from_directory
 import tzlocal
+from flask import Blueprint, Flask, request, send_from_directory
```

Comment:
How come these were added?

Reply:
These were here from the beginning. Pre-commit just formatted it and move it to another line